kratos/sources/model_part_io.cpp

```diff
@@ -220,6 +220,105 @@ void ModelPartIO::WriteProperties(PropertiesContainerType const& rThisProperties
     }
 }
 
+void ModelPartIO::ReadGeometry(NodesContainerType& rThisNodes, GeometryType::Pointer& pThisGeometries)
+{
+    KRATOS_ERROR << "Calling base class member. Please check the definition of derived class" << std::endl;
+}
+
+void ModelPartIO::ReadGeometries(NodesContainerType& rThisNodes, GeometryContainerType& rThisGeometries)
```

Comment:
```suggestion
void ModelPartIO::ReadGeometries(
    NodesContainerType& rThisNodes,
    GeometryContainerType& rThisGeometries)
```
(minor style guide comment)